.env

```diff
@@ -35,6 +35,9 @@ SMTP_USERNAME = ""
 SMTP_PASSWORD = ""
 SMTP_EMAIL = ""
 
+# Push notifications using Firebase Cloud Messaging
+USE_FIREBASE=false
```

Comment:
Should not this be true, since we want notification

Reply:
By default, notifications should be disabled. We will enable them only on the prod and preprod.